gradle.properties

```diff
@@ -12,7 +12,7 @@ kotlin.mpp.enableCInteropCommonization=true
 kotlin.mpp.stability.nowarn=true
 kotlin.native.ignoreDisabledTargets=true
 
-artifactVersion = 5.2.2
+artifactVersion = 5.2.3-SNAPSHOT
```

Comment:
SNAPSHOT?

Reply:
I'd changed the version number after succesful review, but since you're insisting ...